### DISCUSSION TITLE
New Channels to older List

### DISCUSSION THREAD

**Discussion by TsinJu** [PredatH0r/ChanSort#308] [Q&A]
I have created a new channel list, my LG TV has now detected some channel changes/additions and would like me to carry out a new channel search. 
Now I don't want to have to sort all the channels again. 
Can both channel lists (old/new) be loaded into Chansort in order to then transfer the new channels to the old channel list?

Thx for helping

**Comment by PredatH0r** [comment]
Keep your old channel list somewhere on your PC.
Export the new channel list from your TV to a USB Stick.
Open the new list with ChanSort. It will automatically open a popup offering an option to apply the same ordering as in another list.
Choose that option and then select your old list in the next step.

Since both lists are from the same TV, you should be able to select "automatic" matching between sub-lists (antenna, cable, satellite) and press "Apply".
After that, your new list should have the same order as the old one. Any missing channels are shown in red.
